FWCore/Framework/src/ESProducer.cc

```diff
@@ -2,7 +2,7 @@
 //
 // Package:     Framework
 // Class  :     ESProducer
-// 
+//
```

Comment:
It looks like you only changed spaces here? Can this be reverted back?

Reply:
Yikes, yes that's true.  Will revert.